src/PowerShellEditorServices/Language/LanguageService.cs

```diff
@@ -475,6 +487,20 @@ public FindOccurrencesResult FindSymbolsInFile(ScriptFile scriptFile)
                 null;
         }
 
+        /// <summary>
+        /// Gets a path from a dot-source symbol.
+        /// </summary>
+        /// <param name="symbol">The symbol representing the dot-source expression.</param>
+        /// <param name="workspace">The current workspace</param>
+        /// <param name="scriptFile">The script file containing the symbol</param>
+        /// <returns></returns>
+        private static string GetDotSourcedPath(SymbolReference symbol, Workspace workspace, ScriptFile scriptFile)
+        {
+            string cleanedUpSymbol = PathUtils.NormalizePathSeparators(symbol.SymbolName.Trim('\'', '"'));
+            return workspace.ResolveRelativeScriptPath(Path.GetDirectoryName(scriptFile.FilePath),
```

Comment:
Might be worth factoring out the `Path.GetDirectoryName(scriptFile.FilePath)` as a variable here -- saves an allocation at least.